### PR TITLE
First crack at property order reasoning

### LINKED ISSUE #59: Describe reasoning for CSS property order
CSS property order guidelines should explain the reasoning for the recommendation

### PR DESCRIPTION
**Pull request by Scotchester**
Let me know how this reads.

There's one property not shown here that I wonder about which group it belongs in: `content`. This seems like a significant/non-decorative enough property to not go in the "other" bucket. Should it be listed in the first group?

Any other ambiguous properties anyone else wonders about?

Closes #59

### CODE REVIEW

File: css.md [hunk 1]
```diff
@@ -23,17 +23,29 @@ such that it's highly readable and consistent across different developers on a t
...
-- 
 
 ## Property Order
 
-Box model properties go first, in order from inside the box out, followed by positioning.
+Declare properties in three groups:
 
-Remaining properties are in alphabetical order.
+1. Display & box model
+   1. Switching `display` and/or `box-sizing`
```
Comment (ascott1) [inline css.md]: I'm a little confused by the use of "switching" here. Do you mean changing the default display or box-sizing values?

**Comment by ascott1** [comment]
Other than the comment above this looks great. Sorry for sitting on this for so long @Scotchester.

File: css.md (hunk 1 above)
Comment (Scotchester) [inline css.md]: Yes, that's what I meant. You're right, it could use clarification. I'll
update it.

Comment (Scotchester) [inline css.md]: Updated in 5f740b1305534d470ec9f545cdecf89687d13812

**Comment by ascott1** [comment]
:+1: LGTM. Thanks @Scotchester!

**Pull request merged**